### ISSUE TITLE
Typescript compile errors in default electron.ts

### ISSUE THREAD

**Issue by francovp** [BenjaminDobler/ngtron#12]
I got this kind of errors from typescript compiler in `src/electron.ts`:

```
Error:(3, 9) TS2305: Module '"../../../../../Users/MyUser/my-project/src/electron"' has no exported member 'BrowserWindow'.
Error:(3, 24) TS2305: Module '"../../../../../Users/MyUser/my-project/src/electron"' has no exported member 'app'.
```

Any ideas why?

Mi environment:

```
Angular CLI: 8.3.6
Node: 12.9.1
OS: win32 x64
Angular: 8.2.2
... animations, common, compiler, compiler-cli, core, forms
... platform-browser, platform-browser-dynamic, router
... service-worker

Package                            Version
------------------------------------------------------------
@angular-devkit/architect          0.802.2
@angular-devkit/build-angular      0.802.2
@angular-devkit/build-optimizer    0.802.2
@angular-devkit/build-webpack      0.802.2
@angular-devkit/core               8.2.2
@angular-devkit/schematics         8.2.2
@angular/cdk                       7.3.7
@angular/cli                       8.3.6
@angular/fire                      5.2.1
@angular/flex-layout               8.0.0-beta.26
@angular/material                  7.3.7
@angular/material-moment-adapter   7.3.7
@angular/pwa                       0.13.9
@ngtools/webpack                   8.2.2
@schematics/angular                8.2.2
@schematics/update                 0.803.6
rxjs                               6.5.2
typescript                         3.5.3
webpack                            4.38.0
```

**Comment by francovp** [comment]
Additionally, when I run `ng run my-project:build-electron` I got this in the output:

```
Date: 2019-10-01T19:20:30.280Z
Hash: 1a3089d73249fe708a6d
Time: 9440ms
chunk {main} main.js (main) 604 bytes [initial] [rendered]
chunk {polyfills} polyfills.js (polyfills) 138 kB [initial] [rendered]
chunk {runtime} runtime.js (runtime) 6.04 kB [entry] [rendered]
chunk {scripts} scripts.js (scripts) 425 kB [entry] [rendered]
chunk {styles} styles.js (styles) 495 kB [initial] [rendered]

ERROR in No NgModule metadata found for 'AppModule'.
```

**Comment by MaGnaL** [comment]
The latter is also the case with me.

Angular 8.2.8
CLI: 8.3.6

**Comment by BenjaminDobler** [comment]
It looks like the BrowserWindow and app are wrongly imported from the electron.ts in src and not from the electron module. Is this a blank setup or have you added any code already?

**Comment by BenjaminDobler** [comment]
If you do a simple project like this:
ng new ngtron-example
ng add @richapps/ngtron
ng run ngtron-example:build-electron

Do you get these errors?

**Comment by MaGnaL** [comment]
> If you do a simple project like this:
> ng new ngtron-example
> ng add @richapps/ngtron
> ng run ngtron-example:build-electron
> 
> Do you get these errors?

Yes. That's exactly what I did. But I do not have the errors @francovp mentioned in his first post.

**Comment by BenjaminDobler** [comment]
@MaGnaL Any chance you could provide me your project?

**Comment by MaGnaL** [comment]
Sure: https://github.com/MaGnaL/ngtron-reprodocution

Steps to reproduce:
- `npm i`
- `ng run ngtron-test:build-electron`
- you get this:
![image](https://user-images.githubusercontent.com/4709544/66026529-51b02a00-e4f9-11e9-91b6-532935dee2dc.png)

Btw: The standard `ng serve` and `ng build` works.

**Comment by BenjaminDobler** [comment]
@MaGnaL Thanks a lot! I was able to track it down. I wrongly have an older version of the @angular-devkit/build-angular package in my dependencies which shouldn`t be there at all. 
If you go into the @richapps/ngtron/node_modules folder and remove @angular-devkit/build-angular and @ngtools it should build correctly. 
I will update ngtron later today. One other thing i noticed. It seems like electron is not playing nicely with the new differential loading feature of Angular 8. So you will need to set the target in your tsconfig to es5 for now. 
Let me know if that works for you and thanks for your help!

**Comment by MaGnaL** [comment]
Yes. I can confirm, that it works with those changes. :+1: Thanks for your effort.

@francovp maybe this will fix the problem in your first post as well?

**Comment by BenjaminDobler** [comment]
Released a small update 0.0.25 with the dependency fix.

**Comment by MaGnaL** [comment]
Works with 0.0.25.
